sys-apps/less/less-643.recipe

```diff
@@ -48,18 +48,23 @@ BUILD()
 {
 	autoconf
 	runConfigure ./configure --with-editor=nano \
```

Comment:
```suggestion
	runConfigure --omit-dirs binDir ./configure --bindir $prefix/bin --with-editor=nano \
```

Reply:
Nitpicking here ....
We introduced (don't know who started it) :) commandBinDir for these cases, if we stop using these we might as well drop them in the future ...

Reply:
I think `$commandBinDir` was intended for more complicated cases than just secondaryArch on x86_gcc2 (maybe for future 32-bit support on x86_64? I don't know).

This was just a "quick and dirty" suggestion with the minimal changes to get it working correctly on x86. Using `$commandBinDir` is probably cleaner.

Reply:
The situation for 64bit secondary architecture would be the same as for 32bit I guess, so I don't think that would explain it (I know I got corrected when not using it in the past) :)